CovertInfiltration/Config/XComMissionSources.ini

```diff
@@ -1,42 +1,48 @@
++ActivityMissionFamily=(ActivityTemplate="Activity_IntelligenceAssault", MissionFamily="ProtectDevice")
++ActivityMissionFamily=(ActivityTemplate="Activity_InformantInfiltrate", MissionFamily="Rescue")
++ActivityMissionFamily=(ActivityTemplate="Activity_InformantInfiltrate", MissionFamily="Extract")
++ActivityMissionFamily=(ActivityTemplate="Activity_InformantInfiltrate", MissionFamily="Compound")
++ActivityMissionFamily=(ActivityTemplate="Activity_InformantAssault", MissionFamily="Extract")
++ActivityMissionFamily=(ActivityTemplate="Activity_InformantAssault", MissionFamily="RecoverExpedition")
++ActivityMissionFamily=(ActivityTemplate="Activity_InformantAssault", MissionFamily="SwarmDefense")
+
+; soldier or staff reward
++ActivityMissionFamily=(ActivityTemplate="Activity_PersonnelRescue", MissionFamily="Rescue")
++ActivityMissionFamily=(ActivityTemplate="Activity_PersonnelRescue", MissionFamily="Extract")
++ActivityMissionFamily=(ActivityTemplate="Activity_PersonnelRescue", MissionFamily="Compound")
+
+; region income reward
++ActivityMissionFamily=(ActivityTemplate="Activity_DistractionInfiltrate", MissionFamily="SabotageCC")
++ActivityMissionFamily=(ActivityTemplate="Activity_DistractionAssault", MissionFamily="SabotageCC")
+
+; delay facility reward
+; overridden by intercept chain to reward the lost materials
++ActivityMissionFamily=(ActivityTemplate="Activity_SabotageInfiltrate", MissionFamily="DestroyObject")
++ActivityMissionFamily=(ActivityTemplate="Activity_SabotageInfiltrate", MissionFamily="SabotageTransmitter")
++ActivityMissionFamily=(ActivityTemplate="Activity_SabotageInfiltrate", MissionFamily="SabotageV2")
++ActivityMissionFamily=(ActivityTemplate="Activity_SabotageInfiltrate", MissionFamily="NeutralizeFieldCommander")
++ActivityMissionFamily=(ActivityTemplate="Activity_SabotageAssault", MissionFamily="DestroyObject")
++ActivityMissionFamily=(ActivityTemplate="Activity_SabotageAssault", MissionFamily="SabotageTransmitter")
++ActivityMissionFamily=(ActivityTemplate="Activity_SabotageAssault", MissionFamily="NeutralizeFieldCommander")
+
+; large supply reward
++ActivityMissionFamily=(ActivityTemplate="Activity_SupplyConvoy", MissionFamily="SupplyLineRaid")
++ActivityMissionFamily=(ActivityTemplate="Activity_SecureUFO", MissionFamily="SecureUFO")
+
+; large supply reward, near impossible to get all
+; also tagged as distraction (assault)
++ActivityMissionFamily=(ActivityTemplate="Activity_SupplyExtract", MissionFamily="SupplyExtraction")
+
+; POI plus medium intel if captured
+; overridden by informant chain to reward POI and datapad plus high intel if captured
+; also tagged as distraction (infiltrate)
++ActivityMissionFamily=(ActivityTemplate="Activity_CaptureDVIP", MissionFamily="Neutralize")
+
+[XComGame.XComTacticalMissionManager]
++arrSourceRewardMissionTypes=(MissionSource="MissionSource_ActivityCI", RewardType="Reward_Rumor", MissionFamily="ProtectDevice")
```

Comment:
🤔

Reply:
I apparently had to do that or else the game was spitting redscreens at me

Reply:
Probably you did something wrong then. What was the redscreen?

Reply:
Something about a missing quest item I think

Reply:
I suggest that you remove this line and see what the redscreen is again

Reply:
```
[0545.84] NoFullyLoading: Testing: Could not find matching quest item template for:
[0545.84] NoFullyLoading: Testing:    Mission Source: MissionSource_ActivityCI,
[0545.84] NoFullyLoading: Testing:    Mission Reward: None,
[0545.84] NoFullyLoading: Testing:    Mission Type: ProtectDevice
[0545.84] NoFullyLoading: Testing: Using AccessCodes as a fallback, since this mission appears to require a quest item!
```